src/site/twitch.tv/modules/player/PlayerController.vue

```diff
@@ -3,22 +3,25 @@
 		<PlayerContentWarning v-for="x of contentWarning.instances" :key="x.identifier" :inst="x" />
 	</template>
 </template>
-1
 
 <script setup lang="ts">
 import { onUnmounted, ref, watch, watchEffect } from "vue";
 import { debounceFn } from "@/common/Async";
 import { HookedInstance, ReactComponentHook, defineComponentHook, unhookComponent } from "@/common/ReactHooks";
-import { definePropertyHook } from "@/common/Reflection";
+import { defineNamedEventHandler, unsetNamedEventHandler, unsetPropertyHook } from "@/common/Reflection";
 import { useConfig } from "@/composable/useSettings";
 import PlayerContentWarning from "./PlayerContentWarning.vue";
 
 const props = defineProps<{
 	inst: HookedInstance<Twitch.VideoPlayerComponent>;
+	mediaPlayer?: Twitch.MediaPlayerInstance;
 }>();
 
 const shouldHideContentWarning = useConfig<boolean>("player.skip_content_restriction");
+const actionOnClick = useConfig<number>("player.action_onclick");
+
 const contentWarning = ref<ReactComponentHook<Twitch.VideoPlayerContentRestriction> | null>(null);
+const playerOverlay = ref<HTMLElement | null>(null);
 
 props.inst.component.props.containerRef.classList.add("seventv-player");
```

Comment:
Would be nice here to do a check first and log to the console, if the geometry changes here this will fail the whole component. Could be done in a later PR though.